tests/regionsearch/test_rowset.py

```diff
@@ -0,0 +1,104 @@
+import random
+
+from astropy import units as u
+from astropy.coordinates import EarthLocation, SkyCoord
+from astropy.table import Column, Table
+from astropy.time import Time
+
+from kbmod.regionsearch.region_search import RowSet
+
+rowcnt = 10
+random.seed(0)
+
+pointing = SkyCoord(
+    ra=[(42.0 + random.uniform(0, 2)) for i in range(rowcnt)] * u.deg,
+    dec=[(-28.0 + random.uniform(0, 2)) for i in range(rowcnt)] * u.deg,
+    frame="icrs",
+)
+# time = [ Time(f"2023-06-{(1+3*i):02d}T00:00:00.000", format="isot", scale="utc") for i in range(rowcnt) ]
+time = Time([f"2023-06-{(1+3*i):02d}T00:00:00.000" for i in range(rowcnt)], format="isot", scale="utc")
+fov = [0.2 for i in range(rowcnt)] * u.deg
+distance_far = [random.randrange(7, 10000) for i in range(rowcnt)] * u.au
+distance_near = [i - 5 for i in distance_far.value] * u.au
+# distance_near = [ distance_far[i] - 5 ] for i in range(rowcnt) ] * u.au
```

Comment:
Remove commented out code.

Reply:
I complete removed RowSet as it is not used and as you pointed out (later) it has lots of ifs ands or buts.